doc/api/vm.md

```diff
@@ -31,7 +31,7 @@ const x = 1;
 const context = { x: 2 };
 vm.createContext(context); // Contextify the object.
 
-const code = 'x += 40; var y = 17;';
+const code = 'x += 40; const y = 17;';
 // `x` and `y` are global variables in the context.
 // Initially, x has the value 2 because that is the value of context.x.
```

Comment:
If you're using `const` instead of `var`, I don't think it's still a global variable.